packages/core/src/progress-circle/progress-circle.element.scss

```diff
@@ -0,0 +1,109 @@
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/mixins';
+
+:host {
+  --ringcolor: #{$cds-token-color-neutral-200};
+  --fillcolor: #{$cds-token-color-action-800};
```

Comment:
nit: should these have a `-` between words or is this specific to svg?

Reply:
not specific to svg. just missed that. my bad.

Reply:
✅